src/components/ConfirmationBox/ConfirmationBox.js

```diff
@@ -18,6 +18,10 @@ class ConfirmationBox extends React.Component {
     this.props.history.push('/editSnack');
   };
 
+  componentDidMount() {
+    this.itemImage = this.props.storeList[this.props.id].image;
```

Comment:
We can just define this in render, the initial comment about extracting to a function was that it would be beneficial if we wanted to memoise it if values were changed a lot, that doesn't seem to be the case

```
render() {
  const { image } = this.props.storeList[this.props.id]
```